apps/stress-test/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@fluentui/stress-test",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "start": "cross-env NODE_ENV=development webpack-dev-server",
+    "dev": "cross-env NODE_ENV=development webpack-dev-server",
+    "build": "cross-env NODE_ENV=production webpack",
+    "serve": "http-server ./dist",
+    "bench:simple-stress:build": "node ./benchmarks/simple-stress/buildConfig.js",
```

Comment:
instead of so many package.json scripts, it would make sense to encapsulate them into our own CLI API powered by `yargs`

Reply:
I like this idea. Part of my next phase of perf work is to start using this app for more test scenarios so I'll make the CLI as part of that.

Follow up issue: #24309